src/ffi.rs

```diff
@@ -552,26 +573,54 @@ pub extern fn quiche_conn_set_session(
     }
 }
 
+#[repr(C)]
+pub struct RecvInfo<'a> {
+    from: &'a sockaddr,
```

Comment:
In `SendInfo`, `sockaddr_storage` is used but here `sockaddr` is used. Any reason for that?

Reply:
The main difference is that `SendInfo` owns the address value, so we need to hold the needed storage in it, while `RecvInfo` contains only a reference/pointer. In practice I don't think it makes any difference, but it seemed that using `struct sockaddr *` instead of `struct sockaddr_storage *` was better on the C side, so this is just consistent with that.